v2/yarpcprotobuf/procedure.go

```diff
@@ -50,17 +50,17 @@ type StreamProceduresParams struct {
 // Procedures builds a slice of yarpc.Procedures.
 // This is used to construct a slice of yarpc.Procedures from the context
 // of a protoc plugin, i.e. protoc-gen-yarpc-go.
-func Procedures(params ProceduresParams) []yarpc.Procedure {
-	procedures := make([]yarpc.Procedure, 0, 2*(len(params.Unary)+len(params.Stream)))
+func Procedures(params ProceduresParams) []yarpc.TransportProcedure {
```

Comment:
For my own understanding: With the introduction of the `EncodingProcedure`, will the `yarpcprotobuf` package also need to support an API for constructing `[]yarpc.EncodingProcedure` later on?

Reply:
Yeah, I believe so. After my next 2 (maybe 3) PRs, yarpcjson will be responsible for creating the []yarpc.EncodingProcedure that the router registers.